src/pages/registration/resource_type_tab/sub_type_forms/artistic_types/music_performance/ConcertModal.tsx

```diff
@@ -118,43 +138,81 @@ export const ConcertModal = ({ concert, onSubmit, open, closeModal }: ConcertMod
   const [removeWorkItemIndex, setRemoveWorkItemIndex] = useState(-1);
   const closeConfirmDialog = () => setRemoveWorkItemIndex(-1);
 
+  const [partOfSeries, setPartOfSeries] = useState(concert?.concertSeries ? true : false);
+
+  const validateForm = (values?: Concert): FormikErrors<Concert> => {
+    if (!values) {
+      return {};
+    }
+
```

Comment:
Lurer litt på om dette er misvisende 🤔 
Hvis `values` er undefined, så ville jeg heller forventet at jeg fikk feilmeldinger tilbake her (siden formen da ikke kan valideres). Så kanskje heller nekte å kalle denne metoden om man ikke har verdier?
```suggestion
  const validateForm = (values: Concert): FormikErrors<Concert> => {
```